app/views/authors/index.html.erb

```diff
@@ -8,25 +8,25 @@
           <thead class="border-b">
             <tr>
               <th scope="col" class="text-sm font-medium text-gray-900 px-6 py-4 border-r">
-                First name
+                <%= t('authors.fname') %>
```

Comment:
the same, use translations for attributes

Reply:
https://github.com/yuliatokaryk/quotes/pull/24/commits/dc2b4fe4ba91acdf6a957323b45be257ea0fe351